packages/insomnia-app/app/ui/components/templating/tag-editor.js

```diff
@@ -671,6 +714,9 @@ class TagEditor extends React.PureComponent<Props, State> {
           activeTagDefinition.args.map((argDefinition: NunjucksParsedTagArg, index) =>
             this.renderArg(argDefinition, activeTagData.args, index),
           )}
+
+        {activeTagDefinition && this.renderActions(activeTagDefinition.actions)}
```

Comment:
I think this should avoid rendering an empty `form-row` if no actions exist, as such you can either do it here or inside `this.renderActions`.

```suggestion
        {activeTagDefinition?.actions?.length && this.renderActions(activeTagDefinition.actions)}
```